mozci/task.py

```diff
@@ -441,6 +442,8 @@ class LabelSummary(RunnableSummary):
 
     label: str
     tasks: List[Task]
+    from_date: str = ""
+    to_date: str = ""
```

Comment:
Maybe these were meant to go in the other commit? They don't appear used here.

Reply:
I have removed them in the second commit.